### PR TITLE
Add longduration job for Hive on VSphere

### PR DESCRIPTION
**Pull request by fxierh**
Hive supports VSphere. We thus add a longduration job for Hive on this platform.

### CODE REVIEW

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-vsphere-ipi-f28-longduration-hive

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-vsphere-ipi-f28-longduration-hive

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-vsphere-ipi-f28-longduration-hive

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-vsphere-ipi-f28-longduration-hive

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-vsphere-ipi-f28-longduration-hive

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-vsphere-ipi-f28-longduration-hive

**Comment by fxierh** [comment]
Test passed ^^. 

@jianping-shu Would you please take a look ? Thanks.

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml [hunk 1]
```diff
@@ -32267,6 +32267,94 @@ periodics:
...
+        secretName: github-credentials-openshift-ci-robot-private-git-cloner
+    - name: manifest-tool-local-pusher
+      secret:
+        secretName: manifest-tool-local-pusher
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: result-aggregator
+      secret:
+        secretName: result-aggregator
```
Comment (jianping-shu) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml:32357]: Shall we add slack notifier here?

Comment (fxierh) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml:32357]: Slack notifiers are very often overwritten by other contributions due to CI tooling limitations. 
I'd suggest not to add them now.

**Comment by fxierh** [comment]
@liangxia Would you please take a look ? Thanks.

**Pull request merged**